README.md

```diff
@@ -389,6 +389,34 @@ So the comparisons in Ruby would be:
 
 The fields do match successfully, but note the the first comparison matches `id` on `id` and `course_id` so the match would be wider than expected in some instances.
 
+## Page Caching
+
+Any page visit in the App will result in a web request event being sent to Big Query. The event is automatically sent by the Controller after action callback `trigger_request_event`. However, cached pages that are served from rack middleware return early and therefore do not execute any actions in the controller. This means that any cached page visits handled by rack middleware do NOT result in a web request event being sent to Big Query.
+
+To overcome this issue the `dfe-analytics` gem allows the sending of web request events from rack middleware, before the cached page is served, through configuration.
+
+If a page is cached by rack middleware, then a custom `rack_page_cached` proc must be defined in `config/initializers/dfe_analytics.rb`, that returns a boolean indicating whether the page is cached by rack.
+
+For example, if a projects uses standard rails page caching, then a custom `rack_page_cached` proc  can be defined in `config/initializers/dfe_analytics.rb` as follows:
+
+
+```ruby
+DfE::Analytics.config.rack_page_cached = proc do |rack_env|
+  def path; File.join(Rails.root, 'public/cached_pages'); end
```

Comment:
Why have a method here, and not a plain expression?

Reply:
The idea here is to lazily set the path. But git website had a tantrum about this line, so I've removed the method in anycase :-)